src/types/index.ts

```diff
@@ -29,15 +29,14 @@ export const settingsSchema = z.object({
     }),
   email: z.string().email(),
   shortBio: z
-    .string({
-      required_error: "Please type a short bio.",
-    })
+    .string()
     .max(150, {
       message: "Short bio must be at most 150 characters.",
     })
     .min(10, {
       message: "Short bio must be at least 10 characters.",
-    }),
+    })
+    .or(z.literal("")),
```

Comment:
you can just call optinal method at the end to make it optional. take a look here

https://zod.dev/?id=optionals